prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -313,10 +279,10 @@ class FHIRRouter(
         else {
             filter.all {
                 FhirPathUtils.evaluateCondition(
-                    CustomContext(bundle, bundle),
+                    CustomContext(bundle, bundle, filterSubstitutionMap),
```

Comment:
We need to pass the variables, so the FHIR Path resolver can use them too just in case.  This will help with complex substitution of string like %`rsext-somesuffix`